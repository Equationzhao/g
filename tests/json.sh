output="$(g -term-width 200 --no-config --no-update --icons --permission --size --group --owner -j tests/test_data)"
echo "$output" | diff - tests/json.stdout
