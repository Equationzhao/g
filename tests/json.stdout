{
	"entries": [
		{
			"icon": "",
			"permissions": "drwxr-xr-x",
			"size": "128.0 B",
			"name": "?.dir"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "21.0 B",
			"name": "   space file "
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "       -",
			"name": "0.1.112"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "       -",
			"name": "0.1.13"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "       -",
			"name": "0.1.2"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "       -",
			"name": "0.2.10"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "       -",
			"name": "0.2.2"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "       -",
			"name": "0.3.0"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "       -",
			"name": "0.3.84"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "       -",
			"name": "1.0.1"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "       -",
			"name": "1.4.1"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "5.0 B",
			"name": "1.txt"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "7.0 B",
			"name": "10.json"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "100.0 KiB",
			"name": "100KiB"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "6.0 B",
			"name": "11.xml"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "6.0 B",
			"name": "12.yml"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "5.0 B",
			"name": "13.md"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "       -",
			"name": "14.gif"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "       -",
			"name": "2.19.1"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "       -",
			"name": "2.9.1"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "3.0 B",
			"name": "2.c"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "5.0 B",
			"name": "3.cpp"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "       -",
			"name": "4.py"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "6.0 B",
			"name": "5.java"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "       -",
			"name": "6.sh"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "6.0 B",
			"name": "7.html"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "5.0 B",
			"name": "8.css"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "4.0 B",
			"name": "9.js"
		},
		{
			"icon": "",
			"permissions": "-rw-r--r--",
			"size": "       -",
			"name": "9.js~"
		},
		{
			"icon": "",
			"permissions": "lrwxr-xr-x",
			"dereference": "/Users/equationzhao/g/tests/test_data/1.txt",
			"size": "5.0 B",
			"name": "1.txt.link"
		}
	]
}
